knack/util.py

```diff
@@ -151,3 +151,18 @@ def todict(obj, post_processor=None):  # pylint: disable=too-many-return-stateme
                   if not callable(v) and not k.startswith('_')}
         return post_processor(obj, result) if post_processor else result
     return obj
+
+
+def isatty(stream):
+    # Code copied from
+    # https://github.com/tartley/colorama/blob/3d8d48a95de10be25b161c914f274ec6c41d3129/colorama/ansitowin32.py#L43-L53
```

Comment:
Can code under BSD-3-Clause  be used in MIT project?

Reply:
![image](https://user-images.githubusercontent.com/5019528/86866641-f18f8980-c103-11ea-9e5f-71468eeb796c.png)
You should not copy code from BSD to MIT

Reply:
Refactored to remove unnecessary code.